compiler/qsc_frontend/src/typeck/rules.rs

```diff
@@ -7,8 +7,9 @@ use super::{
     Error, Table,
 };
 use crate::resolve::{self, Names, Res};
+use core::panic;
```

Comment:
This is a weird side effect of tab completion with rust-analyzer where it will add this, but it turns it out isn't needed since `core` is part of the default Rust prelude.


```suggestion
```

Reply:
removed